src/main/cpp/RLBotInterface/BallPrediction/inc/DynamicState.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#include "linalg.h"
+
+struct DynamicState {
+  vec3 x;
```

Comment:
what do these variables mean?

Reply:
x is position, v velocity, w angular velocity, o orientation. They are similar to the notation I used in my blog posts.

Reply:
80 character line limits make me prefer short names, rather than typing "ball.angular_velocity"